criu/include/sk-inet.h

```diff
@@ -76,7 +76,13 @@ static inline void tcp_repair_off(int fd)
 
 	ret = setsockopt(fd, SOL_TCP, TCP_REPAIR, &aux, sizeof(aux));
 	if (ret < 0)
-		pr_err("Failed to turn off repair mode on socket\n");
+		pr_warn("Failed to turn off repair mode on socket %d: "
+#ifdef CR_NOGLIBC
+			"%d\n", fd, ret
+#else
+			"%m\n", fd
```

Comment:
General rule in CRIU is that we don't allow using %m, please don't. Also there is no point in printing ret as it is always -1.

I would advise dropping this patch.